packages/typo3-guides-cli/src/Repository/GuidesSettingsRepository.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+declare(strict_types=1);
+
+namespace T3Docs\GuidesCli\Repository;
+
+use T3Docs\GuidesCli\Repository\Exception\FileException;
+
+class GuidesSettingsRepository
+{
+    /**
+     * @return int Number of bytes written
+     */
+    public function create(string $filePath, \DomDocument $document): int
+    {
+        // An error is suppressed to avoid PHP warnings when the file does
+        // not exist. As we check the return value later this should be okay.
+        $result = @file_put_contents($filePath, $document->saveXml());
```

Comment:
Can't we use some library for writing files that offers all this?

Reply:
Hmm, I will postpone this PR and handle the XML stuff first (which is far more complex and with more code in the command). Then we will see, if this PR is necessary in that way.